scripts/examples/gn_silabs_example.sh

```diff
@@ -195,7 +195,7 @@ else
                 if [ "$2" = "rs9116" ]; then
                     optArgs+="use_rs9116=true "
                 elif [ "$2" = "SiWx917" ]; then
-                    optArgs+="use_SiWx917=true "
+                    optArgs+="use_SiWx917=true disable_lcd=true use_external_flash=false "
```

Comment:
why lcd must be disable now  ?

Reply:
SPI multiplexing changes has broken because of wifi-sdk ncp interface file is having SPI driver APIs. Earlier LDMA driver APIs were used for spi tranfer. So till that is fixed by platform team , disabled LCD.

Reply:
why is `use_external_flash` not disabled for the RCP ?

Reply:
This change can be added in your RS9116 PR.

Reply:
I'm not convinced adding this here is the best option. You can build our apps without the build script. This should go in a .gn/.gni.

Reply:
Ok, added in silabs_board.gni for 917 ncp board.